src/Behat/Cores/Drupal7.php

```diff
@@ -237,4 +237,27 @@ public function entityDelete($entity_type, $entity_id) {
     return entity_delete($entity_type, $entity_id);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getDblogEventUrl(int $wid) {
+    return url('/admin/reports/event/' . $log->wid, ['absolute' => TRUE]) . "\n";
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getWatchdogLogMessages(int $scenario_start_time, string $show_php_only) {
```

Comment:
- [ ] I suggest you renaming the method to _getDbLogMessages_
- [ ] Instead of adding a flag to show php only, its better to add an array parameter to accept a list of log severity types (by default empty, allowing show every log). That is because not only the php logs matters. Could you make that change? It implies changing also the showDbLog method .